aggregator/src/aggregation/blob_data.rs

```diff
@@ -157,6 +173,9 @@ impl BlobDataConfig {
                 cond.expr()
                     * (1.expr() - is_padding_next.expr())
                     * (preimage_rlc_next - byte_next.expr()),
+                // the boundary count increments, i.e.
+                // - boundary_count_curr == boundary_count_prev + 1
```

Comment:
Should the "- " at the beginning of this line be deleted?

Reply:
My intention was to add a bullet point. Its not a minus sign. I agree it should be removed for clarity.